.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "java.configuration.updateBuildConfiguration": "disabled"
+}
```

Comment:
wow, first time see this icon. New feature in github.

Reply:
I think this is about missing new line :) but i will remove this unneeded file. Thx!